.github/workflows/msbuild.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       run: dotnet tool restore
     - name: Check code style
       working-directory: dafny
-      run: dotnet tool run dotnet-format -w --check Source/Dafny.sln --exclude Dafny/Scanner.cs --exclude Dafny/Parser.cs
+      run: dotnet tool run dotnet-format -s warn -w --check Source/Dafny.sln
```

Comment:
Why did you remove the excludes?

Reply:
I'm trying to somehow fix the errors that only come up in CI. Any suggestions are very welcome.